### DISCUSSION TITLE
Add ranked on osu!lazer when using the classic mod

### DISCUSSION THREAD

**Discussion by yugi1408** [ppy/osu#17364] [Ideas] [answered]
Ok, so just an idea but since the Classic mod in osu! lazer makes the scores work the same as they do in osu! stable, I was thinking it might be a good idea to allow people who use the Classic mod to still be able to get ranked on leaderboards, earn pp, etc. but only if they use the Classic mod, as some people may prefer the overall look and design of osu! lazer but still want to get ranked using the Classic mod.

**Comment by peppy** [comment, marked as answer]
https://osu.ppy.sh/home/news/2022-03-07-community-meetings-recap
